daemon/daemon.go

```diff
@@ -354,6 +354,37 @@ func (daemon *Daemon) restore() error {
 		return fmt.Errorf("Error initializing network controller: %v", err)
 	}
 
+	//In the case of live-restore==true, if a running container exited before
+	//initNetworkController() is completed, the network of the container
+	//will release failure because of daemon.netController==nil. After
```

Comment:
Nit:  suggest minor wording changes.
"In the case of live-restore==true, if a running container exits before initNetworkController() completes, the daemon will fail to call releaseNetwork() for the container because daemon.netController == nil."

Reply:
Thanks, I have updated.